### PR TITLE
Removed "base branch selection strategy" from dbt Cloud docs

### PR DESCRIPTION
**Pull request by vvkh**
The option was removed from UI because we should always use "merge base" for dbt Cloud configs.
image

### CODE REVIEW

**Comment by williebsweet** [comment]
@leoebfolsom can you lead the review on this? I know you've been spending the most time on this topic.

File: docs/integrations/orchestration/dbt_cloud/configuration.md [hunk 1]
```diff
@@ -36,4 +35,4 @@ Complete the configuration by specifying the following fields:
 | Sampling confidence | The confidence to apply when sampling. |
 | Sampling Threshold | Sampling will be disabled automatically if tables are smaller than specified threshold. If unspecified, default values will be used depending on the Data Source type. |
 
-Click save. Now that you've set up the integration, Datafold will diff your impacted tables whenever you push commits to a PR. A summaryof the diff will appear in GitHub, and detailed results will appear in the Datafold app.
\ No newline at end of file
+Click save. Now that you've set up the integration, Datafold will diff your impacted tables whenever you push commits to a PR. A summaryof the diff will appear in GitHub, and detailed results will appear in the Datafold app.
```
Comment (leoebfolsom) [inline docs/integrations/orchestration/dbt_cloud/configuration.md:38]: @vvkh would you mind fixing this typo that existed before your PR? `summaryof`

**Review by leoebfolsom — APPROVED** [review]
Approved with one tiny requested typo fix.

File: docs/integrations/orchestration/dbt_cloud/configuration.md (hunk 1 above)
Comment (vvkh) [reply docs/integrations/orchestration/dbt_cloud/configuration.md]: ✅

**Pull request merged**